_content/av/back-to-nature_yuttadhamo.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Back to nature"
+authors:
+- yuttadhammo
+external_url: "https://tunein.com/podcasts/Religion--Spirituality-Podcas/Dhamma-Talks-(Part-3)-p2926505/?topicId=220379444"
+source_url: ""
+drive_links:
+  - "https://drive.google.com/file/d/1cmOj8zVQPfMFeLqjq_ugnOf5-bhzoki4/view?usp=sharing"
+formats: [m4a]
+course: nature
+tags:
+  - meditation
+  - city
+  - mind
+  - forest
+year: 2013
+minutes: 41:35
+---
+Description
```

Comment:
Needs a real description :) I like quotes for talks. Perhaps:

```md
> Many people have this paradigm of seeing it all as natural. If you take the Materialist stance, you'd have to say that things like rape and murder are natural [too because] you don't have a clear framework with which to explain morality.
```

Was there a quote that jumped out at you?